tests/test_api.py

```diff
@@ -15,6 +17,7 @@ def test_post(server):
     response = httpx.post(server.url, data=b"Hello, world!")
     assert response.status_code == 200
     assert response.reason_phrase == "OK"
+    assert response.elapsed > timedelta(seconds=0)
```

Comment:
So, now that `elapsed` is set by the client, should we move this to a separate test in `test_client.py` and `test_async_client.py`? I'd even be in favor of a dedicated `test_elapsed.py` test file.

There's a neat `/slow_response` endpoint on our uvicorn test server that we can call and check that:

- We've got a response within the endpoint's sleep time.
- The elapsed time doesn't account for reading the response (e.g. compare the elapsed time before/after reading the response).

Reply:
Done